tools/iree-prof-output.cc

```diff
@@ -22,6 +23,11 @@ ABSL_FLAG(std::string, output_xplane_file, "",
           "Xplane file to write as the output of execution or conversion.");
 ABSL_FLAG(bool, output_stdout, true,
           "Whether to print Tracy result to stdout.");
+ABSL_FLAG(std::vector<std::string>, zone_substrs, {"iree_hal_buffer_map_"},
```

Comment:
Later we may want to add a file support from which we get filters. But it is okay for now.

Reply:
Ack